docs/faq.rst

```diff
@@ -391,6 +391,17 @@ How can I configure and debug the exact version exported by my package?
 
     (This requires the ``setuptools-scm`` package is installed in your environment)
 
+Why do I see the error ``AssertionError: cant parse version ...``?
+    There are some limitations with ``setuptools-scm`` and in some edge cases
+    it might fail to parse specific tags.
+    If that is happening to your project (other than removing the problematic tag),
+    you can try to re-run the relevant command after setting the
+    ``SETUPTOOLS_SCM_PRETEND_VERSION`` environment variable to overwrite the
+    default behavior.
```

Comment:
Thank you, that is a good suggestion.

```suggestion
    default behavior.
    For example, if you are using bash in your terminal, you can run
    `export SETUPTOOLS_SCM_PRETEND_VERSION=v0.42`
    before running the tox commands.
```

This reminds me that we should add `SETUPTOOLS_SCM_PRETEND_VERSION` to the `passenv` configuration in the tox file to allow this workaround to work... What do you think?

Reply:
Cool idea!